docs/maintaining.md

```diff
@@ -63,31 +63,59 @@ docker compose --profile full_stack up -d
 ```
 
 !!! note "Specify the deployment profile"
-    
+
     Whenever you use `docker compose` commands, you also have to specify
     the deployment profile you want to use with the `-p` or `--profile` flag.
     If you forget this step, `docker compose` will use the default profile (`local_node`),
     and will only shut down, update, or start the services for that profile.
 
 ## Updating the data in your graph
 
+If you have followed the [initial setup](getting_started.md)
+and have deployed your Neurobagel node from our Docker Compose recipe,
+your node will have a dedicated graph database that only contains 
+the data for your node.
+
+This makes updating the data in your graph a straightforward process.
+Once you have generated the updated files you want to upload,
+the process to update the data in your graph is:
+
+1. Shut down the Neurobagel node
+
+    ```bash
+    docker compose --profile full_stack down
+    ```
+
+2. Update the data files in [your `LOCAL_GRAPH_DATA` directory](config.md#uploading-data-to-the-graph-store)
```

Comment:
I find it kind of confusing to have to navigate to a section under the "Manual setup" steps just to see how to change the default toy data, and in this case I think it's also not necessary. How about just directly stating what they need to change in the .env file:

```suggestion
2. In your `.env` file, replace the value of the `LOCAL_GRAPH_DATA` variable with the path to the directory containing your JSONLD files.
```

I think we can further expand on this section a bit in:
- https://github.com/neurobagel/documentation/issues/202

so that we do not really need to refer to the section `config.md#uploading-data-to-the-graph-store` (and just have it for reference, as with the other manual setup steps)

Reply:
What do you think about applying the au